### PR TITLE
Add logging proxy task.

### PR DESCRIPTION
**Pull request by KonovalovaVV**

### CODE REVIEW

File: LoggingProxy/LoggingProxy/LoggingProxy.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            foreach(var member in members)
+            {
+                if(!Members.TryGetValue(member.Name, out object value))
+                    Members.Add(member.Name, member);
+            }
+        }
+
+        public override bool TryInvokeMember(InvokeMemberBinder binder, object[] args, out object result)
+        {
+            SimpleLogger logger = new SimpleLogger();
```
Comment (MaksHladki) [inline LoggingProxy/LoggingProxy/LoggingProxy.cs]: Please, initialize an instance of the SimpeLogger in the constructor.

File: LoggingProxy/LoggingProxy/LoggingProxy.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            _processedObject = processedObject;
+            MemberInfo[] members = typeof(T).GetMembers();
+            foreach(var member in members)
+            {
+                if(!Members.TryGetValue(member.Name, out object value))
+                    Members.Add(member.Name, member);
+            }
+        }
+
+        public override bool TryInvokeMember(InvokeMemberBinder binder, object[] args, out object result)
```
Comment (MaksHladki) [inline LoggingProxy/LoggingProxy/LoggingProxy.cs]: Please, add a reaction to getters and setters `TryGetMember` and `TrySetMember`, In addition, you can add support for events and indexers.

Comment (MaksHladki) [inline LoggingProxy/LoggingProxy/LoggingProxy.cs]: Use try/catch bock inside.

File: LoggingProxy/LoggingProxy/LoggingProxy.cs [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        {
+            SimpleLogger logger = new SimpleLogger();
+            logger.Logger.Warning(binder.Name);
```
Comment (MaksHladki) [inline LoggingProxy/LoggingProxy/LoggingProxy.cs]: `Info` is more relevant here.

File: LoggingProxy/LoggingProxy/LoggingProxy.cs (hunk 1 above)
Comment (KonovalovaVV) [reply LoggingProxy/LoggingProxy/LoggingProxy.cs]: Ok.

File: LoggingProxy/LoggingProxy/LoggingProxy.cs (hunk 2 above)
Comment (KonovalovaVV) [reply LoggingProxy/LoggingProxy/LoggingProxy.cs]: Add.

Comment (KonovalovaVV) [reply LoggingProxy/LoggingProxy/LoggingProxy.cs]: Ok.

File: LoggingProxy/LoggingProxy/LoggingProxy.cs (hunk 3 above)
Comment (KonovalovaVV) [reply LoggingProxy/LoggingProxy/LoggingProxy.cs]: Changed.

**Pull request merged**